spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -253,3 +370,52 @@ definitions:
       - iode:
           type: u8
           desc: Issue of ephemeris data
+
+ - PackedObsContentDeprecatedAlpha:
```

Comment:
From the PR description:

Note a couple of things about deprecated messages: they're not in the deprecated package because sub structures make it non-feasible to move around (see #184). Also, since there can be multiple message deprecations, there's now a naming scheme for deprecations: `DEPRECATED_<GREEK_LETTER>.`

What do you think?